libr/bin/p/bin_dex.c

```diff
@@ -1190,14 +1207,24 @@ static const ut8 *parse_dex_class_method(RBinFile *bf, RBinDexClass *c, RBinClas
 		eprintf ("This DEX is probably corrupted. Chopping DM from %d to 4KB\n", (int)DM);
 		DM = 4096;
 	}
+	size_t skip = 0;
+	ut64 bufsz = r_buf_size (bf->buf);
+	r_buf_seek (bf->buf, offset, R_BUF_SET);
 	for (i = 0; i < DM; i++) {
-		encoded_method_addr = p;
-		ut64 MI, MA, MC;
-		p = r_uleb128 (p, p_end - p, &MI);
+		ut64 encoded_method_addr = offset; //  + skip;
+		bool err = false;
+		skip = 0;
+		r_buf_seek (bf->buf, offset, R_BUF_SET);
```

Comment:
i do not think it is necessary to do this. It seems to me there are no reads in the middle that would change the seek.